.circleci/config.yml

```diff
@@ -514,16 +503,7 @@ workflows:
             packages/eslint-config-audius/.* run-web-workflow true
             packages/eslint-config-audius/.* run-mobile-workflow true
             packages/eslint-config-audius/.* run-harmony-workflow true
-            packages/libs/.* run-creator-workflow true
-            packages/libs/.* run-identity-workflow true
```

Comment:
libs changes a lot. let's not run all the things...

Reply:
I would generally agree - though let's leave these for one of these two workflows. There's often times where a libs change breaks mobile or something else we forget to update

Reply:
added a couple back